examples/ssr/server.js

```diff
@@ -18,6 +18,7 @@ function renderFullPage(html, css) {
         ${css}
         <meta name="viewport" content="initial-scale=1, width=device-width" />
         <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
+        <meta name="emotion-insertion-point" content="" />
```

Comment:
```suggestion
        <meta name="viewport" content="initial-scale=1, width=device-width" />
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
        <meta name="emotion-insertion-point" content="" />
        ${css}
```

One small detail, so that there are no surprises in SSR rendering (this way the styles are always after the stylesheet for the icons in inserted, which is correct as some overrides can be added for the icons.